pkg/txn/storage/memorystorage/memorytable/atomic.go

```diff
@@ -0,0 +1,35 @@
+// Copyright 2022 Matrix Origin
```

Comment:
This this file really necesary?   Can we just use atomic.Value?

If this is deemed really necessary, should we move this to common?

Reply:
It's a type-safe wrapper of atomic.Value, the compiler can do type-checks to avoid misuses, and save some type-assertion codes.
I think moving to common is OK.